protocol/storage/fault_slasher.go

```diff
@@ -108,8 +108,8 @@ func (sfm *FaultSlasher) Slash(ctx context.Context, currentHeight *types.BlockHe
 	sfm.log.Debugf("there are %d late miners\n", len(*lms))
 
 	// Slash late miners.
-	for minerStr, state := range *lms {
-		if _, ok := sfm.slashed[minerStr]; ok {
+	for lateMinerActor, state := range *lms {
+		if _, ok := sfm.slashed[lateMinerActor]; ok {
```

Comment:
nit: renaming ok to `alreadySlashed` or something like that would make this already clear code crystal clear.

Reply:
imo the past tense of slash -> slashed is already clear, if this is a strong opinion let me know and I'll rename, else im going to leave it.